src/main/java/nextstep/subway/line/dto/SectionRequest.java

```diff
@@ -0,0 +1,28 @@
+package nextstep.subway.line.dto;
+
+public class SectionRequest {
+	private Long upStationId;
+	private Long downStationId;
+	private int distance;
```

Comment:
Bean Validation을 사용해보셔도 좋을 것 같아요~

```suggestion
      @NotNull
	private Long upStationId;
      @NotNull
	private Long downStationId;
      @NotNull
	private int distance;
```

Reply:
넵. 입력 받을 때 널 체크를 따로 해주지 않아도 되어 좋은 것 같습니다 : )
추가로 
https://bottom-to-top.tistory.com/14
이런 링크도 있네요 @NotNull 과 nullable = false 의 차이점에 대한 내용인데 공부가 되네요 ㅎㅎ
감사합니다 : )